doc/users/explain/event_handling.rst

```diff
@@ -85,25 +85,117 @@ Event name             Class            Description
-   Shift+a          shift, A                      shift, A                       shift, A                       shift, A                       shift, A                       shift, A
-   CapsLock+Shift+a caps_lock, shift, A           caps_lock, shift, A            caps_lock, shift, a            caps_lock, shift, a            caps_lock, shift, a            caps_lock, shift, A
-   Ctrl+Shift+Alt   control, ctrl+shift, ctrl+alt control, ctrl+shift, ctrl+meta control, ctrl+shift, ctrl+meta control, ctrl+shift, ctrl+meta control, ctrl+shift, ctrl+meta control, ctrl+shift, ctrl+alt+shift
-   Ctrl+Shift+a     control, ctrl+shift, ctrl+A   control, ctrl+shift, ctrl+A    control, ctrl+shift, ctrl+A    control, ctrl+shift, ctrl+A    control, ctrl+shift, ctrl+a    control, ctrl+shift, ctrl+A
-   F1               f1                            f1                             f1                             f1                             f1                             f1
-   Ctrl+F1          control, ctrl+f1              control, ctrl+f1               control, ctrl+f1               control, ctrl+f1               control, ctrl+f1               control, Nothing
-   ================ ============================= ============================== ============================== ============================== ============================== ===================================
+   .. tab-set::
+
+      .. tab-item:: Tkinter
+
+         ======================== ===================================
+         Key(s) pressed           Received key event(s)
+         ======================== ===================================
+         :kbd:`Shift+2`           shift, @
+         :kbd:`Shift+F1`          shift, shift+f1
+         :kbd:`Shift`             shift
+         :kbd:`Control`           control
+         :kbd:`Alt`               alt
+         :kbd:`AltGr`             iso_level3_shift
+         :kbd:`CapsLock`          caps_lock
+         :kbd:`CapsLock+a`        caps_lock, A
+         :kbd:`a`                 a
+         :kbd:`Shift+a`           shift, A
+         :kbd:`CapsLock+Shift+a`  caps_lock, shift, a
+         :kbd:`Control+Shift+Alt` control, ctrl+shift, ctrl+meta
+         :kbd:`Control+Shift+a`   control, ctrl+shift, ctrl+a
+         :kbd:`F1`                f1
+         :kbd:`Control+F1`        control, ctrl+f1
+         ======================== ===================================
+
+      .. tab-item:: Qt
+
+         ======================== ===================================
+         Key(s) pressed           Received key event(s)
+         ======================== ===================================
+         :kbd:`Shift+2`           shift, @
+         :kbd:`Shift+F1`          shift, shift+f1
+         :kbd:`Shift`             shift
+         :kbd:`Control`           control
+         :kbd:`Alt`               alt
+         :kbd:`AltGr`             *nothing*
```

Comment:
Are these actually *nothing* or are they *unknown*? For example, I filled out most of the macosx section and I just don't have an `AltGr` key, so I just had no idea what it produces :)

Reply:
I don't know actually, so I've reverted it to be empty.